securitas/templates/group.html

```diff
@@ -13,6 +13,14 @@ <h3>{{ group.name }}</h3>
           {% if group.description %}<div>{{ group.description }}</div>{% endif %}
       </div>
       {{ edit_group_details() if edit_group_details is defined }}
+      {% if current_user.username in group.members %}
+      <form action="{{ url_for('group_remove_member', groupname=group.name) }}" method="post">
+        {{ sponsor_form.hidden_tag() }}
+        <button type="submit" class="btn btn-outline-danger border-0" name="username" value="{{ current_user.username }}" id="leave-group-btn">
```

Comment:
Would it make more sense to have border-1 or whatever the default is? I found it hard to see on my screen. Maybe even a leave icon would be better? Failing that maybe have it on the groups list as a leave icon next to the green member text.

Reply:
ah yeah, just removed the border-0 which was removing the border here. 

working on a PR to clean this page up a bit more, but added the border back in